File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    nucleotide_complements = {
        "G": "C",
        "C": "G",
        "T": "A",
        "A": "U",
    }

    return "".join([nucleotide_complements.get(n) for n in dna_strand])


def to_dna(rna_strand):
    nucleotide_complements = str.maketrans("CGAU", "GCTA")
    
    return rna_strand.translate(nucleotide_complements)

File: javascript/rna-transcription/rna-transcription.js
const dnaTranscription = {
  "G": "C",
  "C": "G",
  "T": "A",
  "A": "U"
}

export const toRna = (dna) =>
  [...dna].map(ch => dnaTranscription[ch]).join("");

// export const toRna = (dna) => {
//   let rna = "";

//   for (const ch of dna)
//     rna += dnaTranscription[ch];

//   return rna;
// };
